pkg

```diff
@@ -15,7 +15,7 @@
 # Files are relative to this script directory.
 SELF="${BASH_SOURCE[0]}"
 cd "$( dirname "${BASH_SOURCE[0]}" )"
-COMPOSE_FILE=./infrastructure/docker/build/docker-compose.yml
+COMPOSE_FILE=infrastructure/docker/build/docker-compose.yml
```

Comment:
Was this not working for some reason?

Reply:
It works fine, it was removed in 2224c62183. Re-added in 570345c6f.